style.css

```diff
@@ -1,3 +1,80 @@
+
+/* Base line  */
 #root {
-  color: red;
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+}
+
+/* Search bar */
+.search-bar-root {
+  margin: 2rem 0 2rem 0.3rem;
+}
+
+
+.search-query {
+  margin-left: 2rem;
+  width: 14rem;
+  padding: 14px;
+  font-size: 18px;
+  border: 1.5px solid black;
+  border-radius: 5px;
+}
+
+.search-bar-root > p {
+  display: inline;
+  margin-left: 2rem;
+}
+
+/* Select dropdown menu */
+.select-query {
+  width: 20rem;
+  padding: 14px;
+  font-size: 18px;
+  border: 1.5px solid black;
+  border-radius: 5px;
+}
+
+/* Card body */
+.card {
+  width: 30%;
+  border: 1.5px solid #000;
+  flex-grow: 1;
```

Comment:
This is causing the last rendered card to expand to 100% width. I'd recommend removing this. It will also make it more pretty when you select a single episode.